workflows/theiacov/wf_theiacov_ont.wdl

```diff
@@ -272,6 +309,9 @@ workflow theiacov_ont {
       }
     }
   }
+  call versioning.version_capture{
```

Comment:
Any particular reason why this task was moved to the end? I suppose it does not really matter/affect things, I'm just curious why this was moved

Reply:
just to make it prettier and mimic the other workflows